spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/ConsumerProperties.java

```diff
@@ -125,5 +127,13 @@ public HeaderMode getHeaderMode() {
 	public void setHeaderMode(HeaderMode headerMode) {
 		this.headerMode = headerMode;
 	}
+
+	public boolean isSupportDeserializationNatively() {
```

Comment:
I.e. we don't want to use the native decoding *and* the codec, that doesn't make sense.